### ISSUE TITLE
Announce Multi-Selection with audible message

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#3696]
Reporting some first feedback from assistive technologies users, see https://make.wordpress.org/accessibility/2017/11/28/screen-reader-user-experience-with-gutenberg/

From a user:
> While selecting and copying text in the blocks strange things happen. The cursor gets lost and I have to click the mouse to return into the text.

While I wasn't able to reproduce the specific issue, I'm guessing this has to do with selecting text across contiguous blocks. To reproduce:
- add 2-3 contiguous paragraphs with some text
- put the cursor in the middle of the first paragraph
- using the keyboard, start selecting text for example using Shift - Right arrow
- when the selection reaches the end of a block, the blocks multi-selection kicks in and you'll end up with two contiguous blocks selected:

![screen shot 2017-11-28 at 14 38 00](https://user-images.githubusercontent.com/1682452/33322676-5dc10024-d44a-11e7-936e-e10dd417f5d5.png)

Nothing of this is announced with some audible message to screen reader users, nor it is (at the moment) documented in any guide line / help.

Any kind of interaction / behavior which is not native (and thus it's not expected) should be announced with an audible message. A visual feedback is not enough for low vision or blind users.

At the very least:
- blocks multi-selection should be well explained in a Gutenberg help guide, I'd like to see some discussion about a comprehensive guide to Gutenberg usage
- when the multi-selection kicks in, it should be announced and should indicate the number of selected blocks; at the moment, there's only a visual indication in the "blocks" tab
- not sure why focus is moved to the settings ellipsis button: this is a not expected behavior and should not happen; it is far better to let users find this button in the natural tab sequence after the multi selection, so I'd propose it should be placed _after_ the multi-selection
- the current button text "Open Settings Menu" is not so meaningful: as this control appears to contain actions that are only available with a multi-selection, it should have a meaningful text to convey this information, something like "Multi selection actions" (better wording welcome)

**Comment by afercia** [comment]
Discussed during today's accessibility meeting, and agreed the part about "when the multi-selection kicks in, it should be announced (with an audible message)" should have high priority.
